app/components/audio/song-control.tsx

```
@@ -22,17 +25,16 @@
               <span className="material-symbols-outlined-filled">skip_next</span>
             </IconSmallButton>
           </div>
           <div className="song-progress md:flex items-center gap-4 hidden">
-            {/* <p>{time}</p> */}
-            {/* <div className="song-progress-bar flex-grow bg-[--md-sys-color-on-surface] h-1 rounded-full overflow-clip">
-              <div className="song-progress-bar-inner bg-[--md-sys-color-primary] h-full rounded-full"></div>
-            </div> */}
-            <input type="range" aria-label="input" className="w-full" />
+            <p>{`${Math.floor(progress / 60)}:${progress % 60 < 10 ? '0' : ''}${progress % 60}`}</p>
+            <input className="w-full" aria-label="song-progress" type="range" value={progress} min={0} max={180} onChange={(e) => {
+              setProgress(parseInt(e.target.value));
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)